cmd/ci_run.go

```diff
@@ -56,20 +52,16 @@ var ciTriggerCmd = &cobra.Command{
 		Runs a trigger for a CI pipeline on the given or current branch if none provided.
 		This API supports variables and must be called with a trigger token or from within GitLab CI.
 
-		Project will be inferred from branch if not provided
-
 		Note: "lab ci trigger" differs from "lab ci create" which is a different API`),
 	Example: heredoc.Doc(`
-		lab ci trigger feature_branch
-		lab ci trigger -p engineering/integration_tests master
-		lab ci trigger -p engineering/integration_tests -v foo=bar master`),
+		lab ci trigger -v foo=bar feature_branch`),
 	PersistentPreRun: labPersistentPreRun,
 	Run: func(cmd *cobra.Command, args []string) {
 		pid, branch, err := getCIRunOptions(cmd, args)
 		if err != nil {
 			log.Fatal(err)
 		}
-		token, err := cmd.Flags().GetString("project")
+		token, err := cmd.Flags().GetString("token")
```

Comment:
So this line definitely looks suspicious, and I'm particularly nervous since I know I tested triggering ci pipelines. It was a manual test however, there isn't automation for that. This does look like a reasonable fix tho.

Reply:
Here's a link to the original PR, I think this indeed was just a bug/copypasta error https://github.com/zaquestion/lab/pull/201/files